release-engineering/role-handbooks/release-image-promotion.md

```diff
@@ -32,12 +32,24 @@ The images are promoted by using the `krel promote-images` command. This command
 
 Depending on the release you're cutting, you need to run the following command:
 
-- If you're cutting Alpha, Beta, or RC release, use the following `krel promote-images` command to promote images for the release you're cutting:
+- If you're cutting Alpha or Beta release, use the following `krel promote-images` command to promote images for the release you're cutting:
 
 ```shell
 krel promote-images -i --fork=<your-github-username> --tag=v1.20.0-beta.1
 ```
 
+- If you're cutting the first RC (e.g. `v1.20.0-rc.0`), you need to promote the images for the RC and for the next minor alpha release (e.g. `v1.21.0-alpha.0`). You can use the following command to that:
+
+```shell
+krel promote-images -i --fork=<your-github-username> --tag=v1.20.0-rc.0 --tag=v1.21.0-alpha.0
+```
+
+- If you're cutting the subsequent RC releases (e.g. `v1.20.0-rc.1`), you only need to promote the images for the RC:
+
+```shell
+krel promote-images -i --fork=<your-github-username> --tag=v1.20.0-rc.1
+```
```

Comment:
I believe that it should be like that, but is that the case actually? I just want to double-check to be sure.

Reply:
This should be correct :)